<?xml version="1.0"?>

<launch>
    
    <arg name="arm_dir" default="left"/>

    <rosparam command="load" file="$(find mission_manage)/params/mission_params.yaml" />

    <include file="$(find ascamera)/launch/hp60c.launch" />

    <node pkg="mission_manage" type="basket_seq_test.py" name="grab_test" output="screen" />    
        
</launch>